github/Requester.py

```diff
@@ -832,6 +832,51 @@ def __structuredFromJson(self, data: str) -> Any:
                     raise
                 return {"data": data}
 
+    def getFile(
+        self,
+        url: str,
+        path: str,
+        parameters: Optional[Dict[str, Any]] = None,
+        headers: Optional[Dict[str, str]] = None,
+        cnx: Optional[Union[HTTPRequestsConnectionClass, HTTPSRequestsConnectionClass]] = None,
+    ) -> None:
+        """
+        GET a file from the server and save it to the given path.
+        """
+        _, _, response = self.getStream(url, parameters, headers, cnx)
+        response.raise_for_status()
+        with open(path, "wb") as f:
+            for chunk in response.iter_content():
+                if chunk:
+                    f.write(chunk)
+
+    def getStream(
+        self,
+        url: str,
+        parameters: Optional[Dict[str, Any]] = None,
+        headers: Optional[Dict[str, str]] = None,
+        cnx: Optional[Union[HTTPRequestsConnectionClass, HTTPSRequestsConnectionClass]] = None,
+    ) -> Tuple[int, Dict[str, Any], requests.Response]:
```

Comment:
Can we return the `response.iter_content()` instead of `response`?

Reply:
Sure, that would mean that `getStream()` should also do `response.raise_for_status()` on the response it gets.